ncm-spma/src/main/perl/spma/yumdnf.pm

```diff
@@ -0,0 +1,91 @@
+use strict;
+use warnings;
+
+package NCM::Component::spma::yumdnf;
+
+use NCM::Component::spma::yum;
+push our @ISA , qw(NCM::Component::spma::yum);
+
+use constant YUM_CONF_FILE => "/etc/dnf/dnf.conf";
```

Comment:
Use read-only instead since this is new code?

Reply:
need to be the same as in the parent `yum.pm`